tests/toitp/CMakeLists.txt

```diff
@@ -42,21 +44,26 @@ foreach(file ${TOITP_TESTS})
   add_dependencies(check_slow ${build_snap})
 
   file(RELATIVE_PATH toitp_test_name ${TOIT_SDK_SOURCE_DIR} ${toitp_test})
-  add_test(
-    NAME ${toitp_test_name}
-    COMMAND $<TARGET_FILE:toit.run> ${toitp_test} ${snap} $<TARGET_FILE:toit.run> ${TOIT_SDK_SOURCE_DIR}/tools/toitp.toit
-    )
-  set_tests_properties(${toitp_test_name} PROPERTIES TIMEOUT 40)
+  if (NOT "${toitp_test_name}" IN_LIST TOIT_SKIP_TESTS)
```

Comment:
continue for skipping?

Reply:
We can't as the snap-test is in the same loop. You could skip the toitp test, and still run the snap test.